assets/pages/support-landing/supportLanding.jsx

```diff
@@ -38,12 +38,20 @@ const supporterSectionId = 'supporter-options';
 const store = pageInit(pageReducer);
 
 
+// ----- Internationalisation ----- //
+
+const CountrySwitcherHeader = CountrySwitcherHeaderContainer(
+  '/contribute',
+  ['GBPCountries', 'UnitedStates', 'EURCountries', 'NZDCountries', 'Canada', 'International', 'AUDCountries'],
+);
+
+
 // ----- Render ----- //
 
 const content = (
   <Provider store={store}>
     <div>
-      <CountrySwitcherHeaderContainer />
+      <CountrySwitcherHeader />
```

Comment:
Probably would be a good idea to rename this variable to `CountrySwitcherHeaderContainer` so the reader knows that we are using a container here.

Reply:
It's imported as `CountrySwitcherHeaderContainer`.